src/src/AdminBundle/Entity/Registration.php

```diff
@@ -184,39 +188,39 @@ public function setAttendee($attendee)
     /**
      * Get the value of eventDetails.
      */
-    public function getEvents()
+    public function getEvent()
```

Comment:
je super, ze si to premenovala ale toto rozbilo polko applikacie.

Reply:
Nerozbilo :) dam na to pozor. Radsej mi povedz, co mam spravit s tym konfliktom.

Reply:
hmmm... rebasni na mastra, vymaz ten file a potom ho vygeneruj nanovo

Reply:
ako sa generuje na novo?